### DISCUSSION TITLE
KATA 2-EJECUTAR ENTORNO VIRTUAL

### DISCUSSION THREAD

**Discussion by AngelTrejoS7** [LaunchX-InnovaccionVirtual/CursoIntroPython#231] [Q&A] [answered]
Buen día 

Pueden mencionarme que errores estoy cometiendo o porque no puedo ejecutar el entrono virtual

Gracias

![Captura de pantalla (2827)](https://user-images.githubusercontent.com/88988612/153804827-fc6a2add-07cc-43ee-bdc1-6c20d00e5d76.png)
![Captura de pantalla (2828)](https://user-images.githubusercontent.com/88988612/153804831-3e841334-5dce-4c20-8e6f-b0a951d47b48.png)

**Comment by OscarDelgad0** [comment]
Hola Angel, checa que cada comando sea ejecutado en el orden correcto y de forma correcta. 
python -m venv env
Ejecuta ese comando para crear un entorno virtual para empezar.
![prueba_1](https://user-images.githubusercontent.com/99148094/153806898-ea776a91-6544-4536-b0d7-6d882d1cd96e.PNG)
Deberías de tener algo así. También te recomiendo que te pases por la sección de discusión y explores los apartados relacionados con tu problema, muchas personas han preguntado sobre esto, puede ayudarte a encontrar respuestas.

**Comment by JavierMendez-Coder** [comment, marked as answer]
Buenas noches, si estas utilizando el sistema operativo Windows el archivo llamado `activate` se deberia encontrar en `env/Scripts/` en vez de en `env/bin/`.

A continuación te dejo un [enlace](https://github.com/JavierMendez-Coder/LaunchX-OnBoarding/tree/master/02-module) mostrando los pasos que realicé al hacer la actividad, por si te resulta útil.

**Reply by carlos-gmx3** [reply]
exactamente, está algo desactualizado el tutorial pero dando click en las carpetas o con el comando tree en la terminal puedes ver todo lo que hay en las carpetas y subcarpetas...

**Comment by doguedogue** [comment]
En windows puedes hacer esto: .\env\Scripts\activate
para salir: deactivate

![image](https://user-images.githubusercontent.com/23409026/153928748-8ceee7f2-261c-4389-aeab-ceb0e353a498.png)
